README.md

```diff
@@ -1,39 +1,25 @@
 # FlutterShow⚡️ - Presentations in Flutter
 
+![FlutterShow Logo](docs/images/fluttershow.png)
```

Comment:
add a new line here, because the badge shows to the right of the image instead of next line
```suggestion
![FlutterShow Logo](docs/images/fluttershow.png)

```